src/Orleans.Runtime/Messaging/OutboundMessageQueue.cs

```diff
@@ -28,6 +28,15 @@ public int Count
             }
         }
 
+        public int ApplicationMessageCount
+        {
+            get
+            {
+                int n = senders.Where(sender => sender.IsValueCreated).Sum(sender => sender.Value.Count);
```

Comment:
Avoid code duplication on line 25 by changing it to:
```csharp
int n = ApplicationMessageCount;
```

Also, it might be cleaner to make it a method instead - `GetApplicationMessageCount()` because it performs some computation, and doesn't simply return a stored value. Similarly, I think `Count` should be refactored to `GetCount()`.